packages/application-components/src/components/tab-list/tab-list.tsx

```diff
@@ -0,0 +1,29 @@
+import type { ReactNode } from 'react';
+
+export type TTabListProps = {
+  /**
+   * Any React node passed to TabHeader.
+   */
+  children: ReactNode;
+  /**
+   * Sets `aria-orientation` attribute. Defaults to `horizontal`
+   */
+  orientation?: 'horizontal' | 'vertical';
```

Comment:
Why do we need this? The `tablist` has the default horizontal orientation already.

Reply:
That is true, however other design system libraries I checked set this explicitly, so I included it. I will delete it.